setup/updating.md

```diff
@@ -28,6 +28,13 @@ helm repo add coder https://helm.coder.com
 Please also ensure that your Kubernetes config is pointing to the cluster on
 which you've deployed Coder.
 
+## Recommendations
+
+- Before beginning, we **strongly recommend** taking snapshots
```

Comment:
I would suggest a phrasing like:

> As with any significant maintenance operation, we recommend taking a snapshot of the database before proceeding with the upgrade. Although Coder tests in-place upgrades and rollbacks involving database state, it is always a good practice to have a recent backup created during a maintenance window immediately prior to the upgrade. It is simpler and safer to roll back directly at the database level, and Coder recommends doing so in the event of an upgrade failure, since it guarantees restoration of the system to a known-working condition.

@coadler is working on our rollback/reversion strategy so may have some more to add in that regard